diff --git forkSrcPrefix/Makefile.libretro forkDstPrefix/Makefile.libretro
index 108148f4579f4727146e5b8c1e05ef7382654208..4ff34d4c2b5c6031ce4c3256f1b70daba99a583c 100644
--- forkSrcPrefix/Makefile.libretro
+++ forkDstPrefix/Makefile.libretro
@@ -417,6 +417,26 @@ else ifeq ($(platform), miyoo)
    CFLAGS += -fno-common -ftree-vectorize -funswitch-loops
    DEFINES += -std=c99
 
+# MY282
+else ifeq ($(platform), my282)
+   TARGET := $(TARGET_NAME)_libretro.so
+   CC = $(CROSS_COMPILE)gcc
+   CXX = $(CROSS_COMPILE)g++
+   AR = $(CROSS_COMPILE)ar
+   SHARED := -shared -Wl,--version-script=link.T
+   fpic := -fPIC
+   PLATFORM_DEFINES += -D_GNU_SOURCE -DHAVE_STRTOF_L -DHAVE_LOCALE -DLOW_LATENCY_AUDIO -DHAVE_FAST_AUDIO -DENABLE_AUDIO_SYNC -DENABLE_FRAMESKIP -DREDUCE_RENDER_FPS -DDISABLE_DEBUG -DREDUCE_CPU_LOAD -DLOW_RES_RENDERING
+   CFLAGS += -O2 -mcpu=cortex-a7 -mfpu=-mfpu=neon-fp16 -mfloat-abi=hard -mtune=cortex-a7 -fno-unroll-loops -fno-lto -fno-tree-vectorize -ffast-math -fno-strict-aliasing -march=armv7-a -mthumb -DNDEBUG -ffunction-sections -fdata-sections
+   LDFLAGS += -Wl,--gc-sections -Wl,-O2 -static-libgcc -static-libstdc++
+   LIBS += -lasound -lpthread
+   
+   
+   
+   
+   
+   
+   
+
 # Windows
 else
    TARGET := $(TARGET_NAME)_libretro.dll
